src/pd/client.rs

```diff
@@ -48,8 +51,10 @@ impl RpcClient {
                     });
                 }
                 Err(e) => {
-                    warn!("validate PD endpoints failed"; "err" => ?e);
-                    thread::sleep(Duration::from_millis(RETRY_INTERVAL_MS));
+                    if i as usize % cfg.retry_log_every == 0 {
```

Comment:
We need to make sure `retry_log_every > 0`.

Reply:
PTAL